packages/entity-browser/src/modules/entityBrowser/reducer.js

```diff
@@ -6,9 +6,26 @@ const showEntityDetail = (state, {payload}) => ({
   showDetailEntityId: payload.entityId
 })
 
+const simpleSearchFieldsToArray = simpleSearchFields => {
+  return simpleSearchFields.split(',')
+    .filter(s => s)
+    .map(s => s.trim())
+}
+
+const simpleSearchFields = (state, {payload}) => {
+  const fieldArray = simpleSearchFieldsToArray(payload.simpleSearchFields)
+  return {
+    ...state,
+    simpleSearchFields: (fieldArray.length === 0) ? ['txtFulltext'] : fieldArray
```

Comment:
This is the same behaviour than we have now. Errors won't be appear in any case. If simple search fields are enabled an no simple search fields are defined in the config `txtFulltext` is the default. But if the search form does not define any txtFulltext field then just no search field will be displayed. I think this is ok.